app/policies/search_policy.rb

```diff
@@ -0,0 +1,15 @@
+# frozen_string_literal: true
+
+# Policies for SearchController
+class SearchPolicy < ApplicationPolicy
+  attr_reader :current_user
+
+  def initialize(current_user, _args = nil)
```

Comment:
`initialize` can be removed completely as it does nothing (setting `@current_user` is not needed in this policy)